src-utils/HydraAuctionUtils/Tx/AutoCreateTx.hs

```diff
@@ -258,5 +258,6 @@ autoSubmitAndAwaitTx ::
   m Tx
 autoSubmitAndAwaitTx params = do
   tx <- autoCreateTx params
+  -- FIXME
```

Comment:
What is this for?

Reply:
It ignores Left, which might be used.

https://github.com/mlabs-haskell/hydra-auction/issues/277